internal/compiler/widgets/fluent/std-widgets.slint

```diff
@@ -90,21 +113,34 @@ SpinBoxButton := Rectangle {
     touch := TouchArea { }
 }
 
-export SpinBox := FocusScope {
+export SpinBox := Rectangle {
+    property <bool> has-focus <=> fs.has-focus;
     property <bool> checked;
     property <int> value;
     property <int> minimum;
     property <int> maximum: 100;
-    property <bool> enabled: true;
+    property <bool> enabled <=> fs.enabled;
     property <image> icon;
     property <length> font-size <=> button.font-size;
 
     min-height: max(32px, l.min-height);
     horizontal-stretch: 1;
     vertical-stretch: 0;
 
-    Rectangle {
-        background: !enabled ? Palette.neutralLighter : Palette.white;
+    background: !enabled ? Palette.neutralLighter : Palette.white;
+
+    fs := FocusScope {
```

Comment:
does clicking on the button put the focus to it? 
The buttons in  https://developer.microsoft.com/en-us/fluentui#/controls/web/button  don't show the focus frame when clicking